Tests/SwerverTests/TestData.swift

```diff
@@ -5,23 +5,23 @@ class TestData {
     static func validUrlOne() -> URL {
         return URL(string: "/")!
     }
-
+    
     static func validUrlTwo() -> URL {
         return URL(string: "/foo")!
     }
-
+    
     static func simpleGetUrl() -> URL {
```

Comment:
I would imagine that most test could be updated to use one of `validUrlOne` or `validUrlTwo` instead of having a bunch of new functions on `TestData`

Reply:
will fix that

Reply:
fixed